### ISSUE TITLE
Lack of debug for HTTP Error

### ISSUE THREAD

**Issue by JohnMcLear** [anxdpanic/plugin.video.twitch#98]
I am getting  `HTTP Error` consistantly when trying to watch a stream.

I am using the latest version of the addon from the git repo.

This is my verboose error log grep'd for "ERROR"

```
23:55:13 T:1728365664   ERROR: CecLogMessage - Write: sent command error !
23:55:13 T:1968050176   ERROR: JSONRPC Server: Failed to connect to sdpd
23:55:14 T:1728365664   ERROR: CecLogMessage - Write: sent command error !
:55:49 T:1728365664   ERROR: CecLogMessage - Write: sent command error !
23:55:49 T:1728365664   ERROR: CecLogMessage - Write: sent command error !
23:55:51 T:1968050176   ERROR: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.twitch/playLive/therattius/]
23:55:51 T:1728365664   ERROR: CecLogMessage - Write: sent command error !
23:55:51 T:1728365664   ERROR: CecLogMessage - Write: sent command error !
```

**Comment by JohnMcLear** [comment]
Updating to latest version of the addon does not resolve this issue.

I think Twitch might have made a change that has nuked your addon functionality.

**Comment by MrSprigster** [comment]
Fixed in #101
